src/python/pants/notes/2.15.x.md

```diff
@@ -1,5 +1,101 @@
 # 2.15.x Release Series
 
+## What's New
+
+Pants 2.15 brings support for several new features:
+
+### Environments
+
+Pants now supports alternate execution "environments" for build actions. These "environments" can be explicitly
+configured as desired via several new target types (including having multiple environments of the same type).
+The new `environment` field on many targets tells Pants which environment to use for a particular target. 
+
+Pants previously supported local execution (which is now configured by the `local_environment` target type) and
+remote execution via the Remote Execution API (which is now configured by the `remote_environment` target type).
+
+New in Pants 2.15 is support for executing build actions within a Docker container configured via the
+`docker_environment` target type.
+
+See the [Environments documentation](https://www.pantsbuild.org/v2.15/docs/environments) for more information.
+
+### Test Batching
+
+`./pants test` now supports batching compatible tests together so that they execute in the same build action.
+This will help to improve the performance of test execution by avoiding the overhead of setting up the
```

Comment:
It's a bit more nuanced than this. It *can* help improve performance, but you're trading off cache hits and possibly concurrency, depending on batch size.

Also, the overhead this helps with is not so much setting up the sandbox, but the test environment more generally, such as setting up initial database state.

How about 

This can improve performance of test execution by sharing test environment setup across multiple test files, but must be balanced against the impact on caching and concurrency.

Reply:
Sounds good, you have the context here so my language will never capture the nuance. I will use your text here.